CHANGELOG.next.md

```diff
@@ -28,6 +28,8 @@ Thanks, you're awesome :-) -->
 
 #### Added
 
+* Added ability to supply free-form usage documentation per fieldset. #988
```

Comment:
Didn't notice earlier, but since we're backporting to 1.7, we should directly add this to the 1.7 changelog.

Reply:
Ah yes - thanks! Will adjust.